classes/target/target_dataroot.php

```diff
@@ -280,7 +280,16 @@ protected function get_target_file_path($file) {
      */
     protected function get_list_of_formats() {
 
-        $availableformats = array('dmY_hi', 'dmy_hi', 'Ydm_hi', 'ydm_hi', 'dmYhi', 'dmyhi', 'Ydmhi', 'ydmhi');
+        $availableformats = array(
```

Comment:
ah, I didn't realist there was already a list of date formats, this is no the normal way of doing things in moodle, normally there is just one date lang key which you can then override the value of